### PR TITLE
Added missing NetworkId to ParticipantData Packet and added toString function for some types

### PR DESCRIPTION
**Pull request by derSoerrn95**

### CODE REVIEW

File: pkg/env/team/constants.go [hunk 1]
```diff
@@ -1,97 +1,198 @@
 package team
 
 // Team IDs
+type TeamsType uint8
```
Comment (anilmisirlioglu) [inline pkg/env/team/constants.go:4]: It would be better to rename it to `Team`

File: pkg/env/session/constants.go [hunk 2]
```diff
@@ -1,18 +1,42 @@
...
+	Q3        SessionType = 7
+	ShortQ    SessionType = 8
+	OSQ       SessionType = 9
+	R         SessionType = 10
+	R2        SessionType = 11
+	R3        SessionType = 12
+	TimeTrial SessionType = 13
 )
+
+var sessionNames = map[SessionType]string{
```
Comment (anilmisirlioglu) [inline pkg/env/session/constants.go:23]: It would be better to rename it to `sessions`

File: pkg/env/driver/constants.go [hunk 3]
```diff
@@ -1,114 +1,232 @@
...
+	LirimZendeli        DriversType = 117
+	DavidBeckmann       DriversType = 118
+	GianlucaPetecof     DriversType = 119
+	MatteoNannini       DriversType = 120
+	AlessioDeledda      DriversType = 121
+	BentViscaal         DriversType = 122
+	EnzoFitipaldi       DriversType = 123
 )
+
+var driverNames = map[DriversType]string{
```
Comment (anilmisirlioglu) [inline pkg/env/driver/constants.go:118]: It would be better to rename it to `drivers`

File: pkg/env/driver/constants.go [hunk 4]
```diff
@@ -1,114 +1,232 @@
 package driver
 
 // Drivers IDs
+type DriversType uint8
```
Comment (anilmisirlioglu) [inline pkg/env/driver/constants.go:4]: It would be better to rename it to `Driver`

File: pkg/env/team/constants.go [hunk 5]
```diff
@@ -1,97 +1,198 @@
...
+	MPMotorsport2021        TeamsType = 111
+	Charouz2021             TeamsType = 112
+	Dams2021                TeamsType = 113
+	Campos2021              TeamsType = 114
+	BWT2021                 TeamsType = 115
+	Trident2021             TeamsType = 116
+	MyTeam                  TeamsType = 255
 )
+
+var teamNames = map[TeamsType]string{
```
Comment (anilmisirlioglu) [inline pkg/env/team/constants.go:101]: It would be better to rename it to `teams`

File: pkg/env/track/constants.go [hunk 6]
```diff
@@ -1,35 +1,74 @@
 package track
 
 // Track IDs
+type TrackType uint8
```
Comment (anilmisirlioglu) [inline pkg/env/track/constants.go:4]: It would be better to rename it to `Track`

**Review by anilmisirlioglu — CHANGES_REQUESTED** [review]

**Comment by derSoerrn95** [comment]
✅

**Comment by anilmisirlioglu** [comment]
Thanks :)

**Comment by derSoerrn95** [comment]
you're welcome :)

**Pull request merged**